ui/packages/components/src/utils/colors.ts

```diff
@@ -0,0 +1,59 @@
+export const rgbToHex = (r: number, g: number, b: number): string =>
+  '#' +
+  [r, g, b]
+    .map((x) => {
+      const hex = x.toString(16);
+      return hex.length === 1 ? '0' + hex : hex;
+    })
+    .join('');
+
+export const resolveColor = (
+  colorValue: string,
+  isDark: boolean,
+  defaultColor: string = '#f6f6f6'
```

Comment:
One thing I find helpful is to keep a comment with the name of the color when we need to add hex values, like  //carbon 50

Reply:
✅